release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2355,8 +2355,8 @@ In the table below, features are marked with the following statuses:
 
 |Memory Manager feature
```

Comment:
Noticed that you removed `feature` in the respective 4.9 PR. https://github.com/openshift/openshift-docs/pull/47624.

For consistency, suggest removing `feature` here.